packages/k6-tdk/src/client/index.ts

```diff
@@ -0,0 +1,39 @@
+import { CookieJar } from 'k6/http';
+
+import { Api } from '@/api';
+import { requestFactory } from '@/utils/http';
+
+import { Account, Adapter, Authenficator, BasicAuth, OpenIDConnect } from '../auth';
+import { Version } from './client';
+import { Resource } from './resource';
+import { Share } from './share';
+import { User } from './user';
+
+
+export { Version } from './client';
+
+export class Client {
+  user: User;
+  share: Share;
+  resource: Resource;
+  constructor(url: string, version: Version, authAdapter: Adapter, account: Account) {
+    let authenficator: Authenficator;
```

Comment:
I suppose? This affects other files as well though, so I'm not sure.

```suggestion
    let authenticator: Authenticator;
```

Reply:
i wrapped my brain around that wording too 🤣 youre correct `authenticator` sounds more pleasant